db/db_mysql/migrations/20160118194630_init.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE IF NOT EXISTS `group_targets` (group_id bigint,target_id bigint );
 CREATE TABLE IF NOT EXISTS `events` (id integer primary key auto_increment,campaign_id bigint,email varchar(255),time datetime,message varchar(255) );
 CREATE TABLE IF NOT EXISTS `campaigns` (id integer primary key auto_increment,user_id bigint,name varchar(255) NOT NULL ,created_date datetime,completed_date datetime,template_id bigint,page_id bigint,status varchar(255),url varchar(255) );
 CREATE TABLE IF NOT EXISTS `attachments` (id integer primary key auto_increment,template_id bigint,content text,type varchar(255),name varchar(255) );
+CREATE TABLE IF NOT EXISTS `imap` (user_id bigint,host varchar(255),port int,username varchar(255),password varchar(255),modified_date datetime,tls boolean,enabled boolean,folder varchar(255,restrict_domain varchar(255),delete_campaign boolean,last_login datetime,last_login_friendly varchar(255)));
```

Comment:
We'll want to move all of these into a new database migration. I'll reach out through the Google group with a note on how to make those using `goose`.

Reply:
Created new migration files with goose 👍